.babelrc

```diff
@@ -0,0 +1,5 @@
+{
+  "presets": [
+    "@babel/preset-env", "@babel/preset-react"
+  ]
+}
```

Comment:
Get outta here newline!

Reply:
I just realized that little red symbol means "no newline at end of file". Shall I add a newline to get rid of it or keep as is? 🥑

Reply:
Just a drive-by review, but I think @jaredkerim means that you should put a newline to get rid of it. I'd also encourage you to see if your text editor has options for automatically ensuring that your files have exactly 1 newline at EOF, making sure no line has trailing whitespace, etc.